tests/shared.py

```diff
@@ -0,0 +1,45 @@
+class MockArgs:
+    def __init__(self, model):
+        self.model = model #part of model config
```

Comment:
can we call `download(...)` here maybe and then directly set `self.train_data = ["./tests/shard_00000000.tar", ]`?

Reply:
or maybe `["./tests/assets/shard_00000000.tar", ]`